docs/user-guide/configuration/workflow.md

```diff
@@ -144,3 +144,16 @@ jobs:
 
 ## Detached Jobs and Pipelines
 It is possible to define workflows that do not have any external trigger. These workflows are "detached" from the normal flow of the pipeline. Some example use cases of this would be to define a rollback flow for your pipeline that could be manually triggered. Invoking a detached pipeline involves the same steps as [Re-running a job's build](../FAQ#how-do-i-re-run-a-jobs-build)
+
+### Example
+In the following example `main` job is detached. All you need to do is omit the `requires` config.
```

Comment:
Actually, this is not entirely correct. For now, they need to have `requires: []` because our current implementation will assume this is triggered by `~pr` or `~commit`
You can plug in the validator to check.

Reply:
If there is only one single job then yes.

Reply:
So this example is not right. We should fix it.

Reply:
We had a user that just ran into this issue so we should definitely add the caveat that you will need to use at least one `requires` keyword in your jobs

Reply:
updated